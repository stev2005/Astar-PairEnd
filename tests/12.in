TCGCCCCGGAGCTGTAGGTCCTACATCTTTCGATTCAAATAGATATTGCGTGCAAGTCCGAGAACTGGAGCAACAAATGCGCATCACACTGGCGGTGACGGGGGTGGTGAAAGATTACAGAGACCGATTCCCTTAGGGCATGACGTATAACATCGATGTGAAATTAGCGTCCCCGACTTCTAGGCAGAGCTGCTGCTAGCTATCCCCGAAAAAGGCGCCAGAT
15
TCCGCTAATTCCAGTCGTCGCTTTACATCTTCAATCTTCTGTATCTAATCGAAGACTAAATTCTTCAG
CCTACGGCGGATAATGAACAAGTTTGGAGCCAGAAAGGCAACTCCGTGTCGTTGTGACTTGATCATCGGGGGTCTATTGGACGTTCCTGAGCACGACTGTCACACCACATTA
CTTGAGCCCGCGAGTTAGACAAGCATCTTCTAAGTACACGGTAGCACCGCGGCCTCAACTCAACTTGATTGCGGTTGACCCTTGATTCTAACCAGATAAGAGTGAGGTGTATGTCTGCAGCATGCCGAGGTGAGACCGATCATATATGCTATTTCAAACGGAATGCATTATCAGCTGACATCTGAATCAATTAAGGCGGAGC
GTGCCGTCCAAACCTGCTCATCGCGATTGAAATTGCCCGTCTTGAGACCGGATACGCCCTCCACAGGGTACATATTGTCTCTTCTATACAATCAAGGGACGCCGCACTAGGCCCGCCCCTCCGTCGC
ACAGGGATGGGGAAAGCGTGAACCGTATCCTCGTAACATAGCGTCGCGCACCAGGGGTGTACCTACTGCTGAAATGTAAAACCAATTGGCCTGGGGTCACAGCCTATGATGACTACGTTCAAAGTGCGTCTAAACTAACATTAAGCCAAACCCCTATAGTGGTGGACTAGT
CAATCCTCGCGTGGTGGCCATAGGATCACATGATCCCCTTGCGATCGGGTGGTCATACTGCTAGGGTTTGGCAACTCACATAGTCGGCTGTACAGTGGTCACGAGTATTCTTCCGGTAAGGTTTTCTCTGGATAACAACTGAGTGAGCCGATCTTCAGGAACATCACCCGCTGTGAAAGCGGAACTCCCTCCACCGG
TTAAGGAATACCCTTCTACACGATAAACGGTGCTGTCGAAGCCTAACAAGAGAACACCCTAACCTACCGCCCGTAGTCGATTGATTAAAGAAGCGGC
TAAACGTCATTTTGGCGCATGACGCTTTCCGACGCGAATATGTGCCTTTTGCATTCTGAATGCAAGGAGGAGAAACCACATTCTTAAGTAGGGTTGCCGATTGATGCAGGCGGGCCTGTGGCACCTT
CGGACAAATGACACTGACTTTGCCACGCCGTGAGGCGTCCCCTCTGATTTGGGTAGAGTCAGTAAGCTCGAGATTTGTGCTAACTATTGGCTAATCGCATTACTACGGAATTATATTTTGCACGAATTCTGCTAATCTCCTCCAACTACGGGGT
TTTGCGAGGAGCCCTTCAACACAAGTGCCCCACCGGTTACTGTCAGACGAGGCGTACCCTTGTA
GGTCGAAATCACCCTCTGTGCTTGAGCGGCGATCCCCCGCTGGAAGGTACCCACTATCGCGACGCGTTACATAGTAACTAGCCGGCGCGCTACAGTTGGTTGGTACTCTTCCATAGAGTGTGGCTCAGTGCGATGAACCTATCAGCGGTG
TAGATTACGGGCGCTGGAACCTGCCTATTGTGGCGGCGTTAGAGTTACAATCTCGACGTAAGGTAACCGAAGGCAGAATACGCAAACCGACTTTGTTTACTATGCTAGTTGAGACGAGTGGACCTTAGGATGCGA
CAAATTCCTGTTAGCGGTACGATCCGTGATACTCGGATATCTGCGTTAGTGCACGCACTAATGTAACATCTACCCTCATTACA
TGCTTATTTCGACAAACAAGCCCGGCGGGGGCCTAATAAGCGGTGT
ATAGACATTCCCTTTCAGCCCCCATTGGGCCGACACTAAGCCACATGCGTGTCTTAGGGATTGACTCATCGCGGGGGATATCTAATATCGTAGGATCCTAGCCAATTGTGGTACTAAATCGTGACGTGAGGGAATATGGTAAGACCCGCATATCAAAAGAGCGGCCGATATAACAGTCAAGTACCCGCCACTCTGGCATACGACACT
