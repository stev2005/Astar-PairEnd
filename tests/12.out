30
52
97
63
81
94
47
61
74
31
72
65
38
22
100
